util-cache/src/main/scala/sbt/util/Output.scala

```diff
@@ -31,3 +32,17 @@ class PlainOutput[J: IsoString](output: OutputStream, converter: SupportConverte
 
   def close() = output.close()
 }
+
+class FileOutput(file: File) extends Output {
```

Comment:
this is new the new Input with a signature similar to `FileInput`, but it still does the intermediary conversion to `String`. However this can still be improved in the future (for example adding it to sjsonnew or if jawn adds serializer capabilities https://github.com/typelevel/jawn/issues/6)

Reply:
I created a PR for sjson-new that would allow us to write directly to a file. https://github.com/eed3si9n/sjson-new/pull/112

Reply:
I have rebased and we now use the methods introduced in `eed3si9n/sjson-new#112`